sources/packages/backend/libs/sims-db/src/entities/student.model.ts

```diff
@@ -110,4 +116,14 @@ export class Student extends RecordDataModel {
     nullable: true,
   })
   disabilityStatusEffectiveDate?: Date;
+
+  /**
+   * Student supplier information data from the integration with Corporate Accounting System (CAS).
+   */
+  @OneToOne(() => CASSupplier, { eager: false, cascade: true, nullable: true })
+  @JoinColumn({
+    name: "cas_supplier_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  casSupplierId?: CASSupplier;
```

Comment:
Please remove the `id` from the relationship, following what we have in the other models.